Data/Sys/Load/GraphicMods/Pandora's Tower/metadata.json

```diff
@@ -0,0 +1,48 @@
+{
+	"meta":
+	{
+		"title": "Bloom and HUD Texture Definitions",
+		"author": "iwubcode"
```

Comment:
I think that's very nice of you to mention me but you can take the credit for this one ;)